dll/win32/shell32/wine/shellole.c

```diff
@@ -810,3 +810,13 @@ HRESULT WINAPI SHCreateQueryCancelAutoPlayMoniker(IMoniker **moniker)
     if (!moniker) return E_INVALIDARG;
     return CreateClassMoniker(&CLSID_QueryCancelAutoPlay, moniker);
 }
+
+/*************************************************************************
+ *  SHIsBadInterfacePtr [SHELL32.84]
+ */
+BOOL WINAPI SHIsBadInterfacePtr(IUnknown *punk, UINT cbVtbl)
```

Comment:
You've put it in a wine-synced file, but this function is not implemented in wine. Please either move it somewhere else, or surround its implementation in `#ifdef __REACTOS__` .

Reply:
Added `#ifdef __REACTOS__` ... `#endif`.

Reply:
Please don't put new functions into wine synced files.

Reply:
Moved to `misc.cpp`.